docs/jade-getting-started.md

```diff
@@ -311,30 +247,171 @@ psql -f db/create-data-repo-db
+* Select test in intellij UI, select 'testConnected' and run or debug it
+
+### 3. Run Integration tests
+We don't recommend running the entire integrated test suite locally, as it takes over two hours to run.
+Instead, you can select a specific test to run either in Intellij or the command line.
+First, make sure you have run through the following steps:
+* Start postgres
+* Ensure docker is running
+* You may need to re-auth with vault every so often. Run `vault login -method=github token=$(cat ~/.gh_token)`
+
+** Run test in the Command Line **
+* Run `source ./render-configs.sh -a integration` to pull secrets from vault, set environment variables and point to Integration Synapse on Azure
+* Start the app locally with `./gradlew bootRun`
+* Open a new command line window, while bootRun runs in the background
+* In the new window, run `source ./render-configs.sh  -a integration` to pull secrets from vault and set environment variables in this context
+* Set this environment variable: `export IT_JADE_API_URL=http://localhost:8080`
+* Run `./gradlew :testIntegration --tests '*testSnapshotBuilder'` to run a specific integration test
+
+** Run or Debug test in Intellij **
+* Run `./render-configs.sh  -a integration -p` to pull secrets from vault and print environment variables
+* Set the environments printed out from the render configs script. You can do this either in
+  your bash profile (you may need to then restart intellij) or in the intellij run configuration for the selected test.
+  * Additionally, set these environment variables:
+  * `export IT_JADE_API_URL=http://localhost:8080`
+* Start application by running `./gradlew bootRun`
+* Select test in intellij UI, select 'testIntegration' and run or debug it
+
+
+### 4. Running Pact tests
+This can be achieved by rendering a small set of Pact-specific configurations first:
+```
+./src/test/render-pact-configs.sh
+# Reload your environment variables, e.g. src ~/.zshrc
+./gradlew verifyPacts     # verify contracts published with TDR as the provider
+```
+
+Note that connected and integration test suites can each take 90+ minutes to run.
+In normal development, you'll likely rely on GitHub Actions / automated PR test runs
+to run all tests, initially running locally those tests which pertain to your work.
+
+To run a subset of tests, you can specify `--tests <pattern>` when running
+the above test commands.  More specific examples are available in
+[Gradle documentation](https://docs.gradle.org/current/userguide/java_testing.html#test_filtering).
+
+### 5. Build `jade-data-repo-ui`
+
+Follow the [setup instructions](https://github.com/DataBiosphere/jade-data-repo-ui#jade-data-repository-ui) to build the `jade-data-repo-ui` repository.
+
+By setting the `PROXY_URL` environment variable, you can point the UI to your local data repo instance.
+```
+export PROXY_URL=http://localhost:8080
+```
+You need to have data repo running with `./gradlew bootRun` and the UI running with `npm start`.
```

Comment:
Do the UI instructions make sense to be here, or should they be in `jade-data-repo-ui`? I would maybe advocate that the UI instructions should be stored primarily in the UI repo, and then there should be a pointer to them from here if we really want it.

Reply:
I think this is just supplemental instructions to that in the UI and there is a link directing the user to the UI readme. And, it's the piece that's relevant to setting up jade-data-repo for running with the UI.